viadot/sources/databricks.py

```diff
@@ -246,6 +248,8 @@ def create_table_from_pandas(
                 Defaults to 'fail'.
             snakecase_column_names (bool, optional): Whether to convert column names to snake case.
                 Defaults to True.
+            cast_df_columns (bool, optional): Whether to cast columns dtypes.
```

Comment:
This doesn't really say anything. Which columns are casted? To which types? Why?